Beta cmdlet docs (#881)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaAdministrativeUnitMember.md

# Add-EntraBetaAdministrativeUnitMember

## Synopsis

Adds an administrative unit member.

## Syntax

```powershell
Add-EntraBetaAdministrativeUnitMember 
 -RefObjectId <String> 
 -ObjectId <String>
 [<CommonParameters>]
```

## Description

> The `Add-EntraBetaAdministrativeUnitMember` cmdlet adds a Microsoft Entra ID administrative unit member. Specify `RefObjectId` and `ObjectId` parameters to add an administrative unit member.

Reviewer: ```suggestion
The `Add-EntraBetaAdministrativeUnitMember` cmdlet adds a Microsoft Entra ID administrative unit member. Specify `RefObjectId` and `ObjectId` parameters to add an administrative unit member.

To add a user, group, or device to an administrative unit, the calling principal must be assigned at least the Privileged Role Administrator Microsoft Entra role.
```

Author: Changes updated.

## Examples

### Example 1: Add an administrative unit member

```powershell

> Connect-Entra -Scopes 'AdministrativeUnit.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All'

Reviewer: ```suggestion
Connect-Entra -Scopes 'AdministrativeUnit.ReadWrite.All'
```

> Add-EntraAdministrativeUnitMember -RefObjectId 'bbbbbbbb-1111-2222-3333-cccccccccccc' -ObjectId 'dddddddd-2222-2222-3333-cccccccccccc'

Reviewer: ```suggestion
$params = @{
    RefObjectId = 'bbbbbbbb-1111-2222-3333-cccccccccccc'
    ObjectId = 'dddddddd-2222-2222-3333-cccccccccccc'
}

Add-EntraAdministrativeUnitMember @params
```

```

> This example demonstrated how to add administrative unit member.

Reviewer: ```suggestion
"This example shows how to add an administrative unit member."
```
